src/Core/src/Handlers/HybridWebView/HybridWebViewHandler.cs

```diff
@@ -30,14 +30,15 @@
 using System.Collections.Specialized;
 using System.Text.Json.Serialization;
 using System.Diagnostics.CodeAnalysis;
+using System.Security.AccessControl;
 
 namespace Microsoft.Maui.Handlers
 {
 	[RequiresUnreferencedCode(DynamicFeatures)]
 #if !NETSTANDARD
 	[RequiresDynamicCode(DynamicFeatures)]
 #endif
-	public partial class HybridWebViewHandler : IHybridWebViewHandler, IHybridWebViewTaskManager
+	public partial class HybridWebViewHandler : IHybridWebViewHandler
```

Comment:
Moved the task manager into a separate service to help with testing ... which I haven't written yet. But also, allows a new handler to work without depending on this internal interface.